src/Agent.Worker/TaskRunner.cs

```diff
@@ -420,7 +420,8 @@ private async Task RunAsyncInternal()
                     }
                 }
 
-                var enableResourceUtilizationWarnings = AgentKnobs.EnableResourceUtilizationWarnings.GetValue(ExecutionContext).AsBoolean();
+                var enableResourceUtilizationWarnings = AgentKnobs.EnableResourceUtilizationWarnings.GetValue(ExecutionContext).AsBoolean()
+                    && !AgentKnobs.DisableResourceUtilizationWarnings.GetValue(ExecutionContext).AsBoolean(); ;
```

Comment:
Maybe this is a stupid question but what's this new knob doing that couldn't be achieved by just setting the EnableResourceUtilizationWarnings knob to false? It seems redundant to have boolean knobs for both EnableResourceUtilizationWarnings and DisableResourceUtilizationWarnings, no?